File: strings/java/src/Data2.java
public class Data2 {

    public static String formatDate(int day, int month, int year) {
        String formatDay = String.format("%02d", day);
        String formatMonth = String.format("%02d", month);

        return formatDay + "/" + formatMonth + "/" + year;
    }

    public static void main(String[] args) {
        System.out.println(formatDate(21, 7, 2010));
    }
}

File: strings/javascript/data2.js
function formatDate(day, month, year) {
  let formatDay = String(day).padStart(2, "0");
  let formatMonth = String(month).padStart(2, "0");

  return `${formatDay}/${formatMonth}/${year}`;
}

console.log(formatDate(21, 7, 2010)); 
